hlapipeline/utils/catalog_utils.py

```diff
@@ -0,0 +1,56 @@
+"""Function to extract random entries (lines) from a CSV file 
+
+The function, randomSelectFromCSV, allows the user to specify the desired
+number of entries to obtain from a comma-separated values (CSV) file
+which contains data extracted from the STScI archive for ACS, WFC3, and 
+WFPC2 instruments. The data are comprised of the following information
```

Comment:
The names of the table columns is actually extraneous for the operation of this specific function, since nothing is done with that information.  It could be removed without affecting the user's understanding of this one function.